index.html

```diff
@@ -189,10 +189,10 @@ <h2 class="text-h2">PDP</h2>
             <div class="footer-list"> 
```

Comment:
add this code to media query max-width 670px
.footer-logo {
        padding-right: 0;
        float: none;
        text-align: center;
    }
    .footer-list li{
        font-size: 18px;
        list-style: none;
        padding-top: 20px;
        padding-right: 60px;
        padding-left: 20px;
        cursor: pointer;
        float : none;
    }
    .call-center{
        padding-right : 0;
        padding-top : 0;
        text-align: center;
    }